pkg/kademlia/kademlia.go

```diff
@@ -121,38 +121,34 @@ func (k *Kademlia) Bootstrap(ctx context.Context) error {
 		return nil
 	}
 
-	var errs errs.Group
+	var errGroup errs.Group
 	for i, node := range k.bootstrapNodes {
 		if ctx.Err() != nil {
-			errs.Add(ctx.Err())
-			return errs.Err()
+			errGroup.Add(ctx.Err())
+			return errGroup.Err()
 		}
 
 		p := &peer.Peer{}
 		pCall := grpc.Peer(p)
 		_, err := k.dialer.PingAddress(ctx, node.Address.Address, pCall)
```

Comment:
`identity, err := k.dialer.PingAddress(`, since it already does it internally, no need for the `pCall` thing.

Reply:
I didn't quite follow, can you elaborate?

Reply:
`PingAddress` already returns the identity, no need to do the peer.Peer thing:

```
p := &peer.Peer{}
pCall := grpc.Peer(p)
```

Reply:
Ah sorry, I meant `FetchPeerIdentity`

Reply:
ah right, it doesn't support arbitrary address... maybe let `PingAddress` return the identity, so the options logic doesn't leak outside?